universe/ValueRefs.h

```diff
@@ -470,7 +470,12 @@ std::string Constant<T>::Description() const
 
 template <typename T>
 void Constant<T>::SetTopLevelContent(const std::string& content_name)
-{ m_top_level_content = content_name; }
+{
+    if (!m_top_level_content.empty())
```

Comment:
can this be just an empty function?

the `std::string` specialization is probably the only one that needs to do anything...

Reply:
ok